Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -106,7 +108,8 @@ open class ComposerVC: _ViewController,
             mentionedUsers: Set<ChatUser>,
             command: Command?,
             extraData: [String: RawJSON] = [:],
-            cooldownTime: Int = 0
+            cooldownTime: Int = 0,
+            skipEnrichUrl: Bool = false
```

Comment:
Shouldn't this be controlled from the dashboard setting, coming in the channel config as `urlEnrichmentEnabled`?

Reply:
No, the `skipEnrichUrl` is only used when `urlEnrichmentEnabled` is true. If you send `skipEnrichUrl = true`, and `urlEnrichmentEnabled` is `false`, it won't enrich any url.

Reply:
We only use it to decide if we should show the link preview or not. If `urlEnrichmentEnabled` is `false` we never show the link preview in the composer. Since when sending a message, the url won't be enriched